model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -288,6 +288,35 @@ class PackageCurationTest : WordSpec({
 
             curatedPkg.pkg.isModified shouldBe false
         }
+
+        "work with Ivy version ranges" {
+            val pkg = Package(
+                id = Identifier(
+                    type = "Maven",
+                    namespace = "androidx.constraintlayout",
+                    name = "constraintlayout",
+                    version = "2.0.3"
+                ),
+                declaredLicenses = sortedSetOf(),
+                description = "",
+                homepageUrl = "",
+                binaryArtifact = RemoteArtifact.EMPTY,
+                sourceArtifact = RemoteArtifact.EMPTY,
+                vcs = VcsInfo.EMPTY
+            )
+
+            val curation = PackageCuration(
+                id = Identifier(
+                    type = "Maven",
+                    namespace = "androidx.constraintlayout",
+                    name = "constraintlayout",
+                    version = "[2.0.1,2.0.4]"
+                ),
+                data = PackageCurationData()
+            )
+
+            curation.isApplicable(pkg.id) shouldBe true
```

Comment:
The test should also verify that a version outside of the range is not applicable.

Reply:
Done.